### PR TITLE
Add documentation for the locale setting

### PR DESCRIPTION
**Pull request by tcitworld**
See https://github.com/nextcloud/server/pull/5623

### CODE REVIEW

File: admin_manual/configuration_server/language_configuration.rst [hunk 1]
```diff
@@ -36,3 +36,34 @@ this value can be set to ``true`` instead of a language code.
...
+
+If you want to force a specific locale, users will no longer be able to change
+their locale in the personal settings. You can set a **force_locale** parameter
+in the :file:`config/config.php`.
+
+'::
+
+    "force_locale" => "en_US",
+
+.. note:: Please check `Transifex language codes
```
Comment (georgehrke) [inline admin_manual/configuration_server/language_configuration.rst]: Copy and Paste leftover? ;) 

What locale should one check? The moment js locales?

Comment (skjnldsv) [reply admin_manual/configuration_server/language_configuration.rst]: Well, the data are provided by moment indeed, so we should probably check moment instead, you're right!

Comment (tcitworld) [reply admin_manual/configuration_server/language_configuration.rst]: Anything better than https://github.com/moment/moment/tree/develop/locale ?

Comment (georgehrke) [reply admin_manual/configuration_server/language_configuration.rst]: I think i would prefer to link to the stable, used version: https://github.com/moment/moment/tree/2.18.1/locale

But then we would have to update the link every time we update moment.js

Comment (tcitworld) [reply admin_manual/configuration_server/language_configuration.rst]: Does it matter much in the end ? Maybe remove the whole note ?

**Pull request merged**